app/views/declaration/goods_measure.scala.html

```diff
@@ -1,74 +0,0 @@
-@*
- * Copyright 2020 HM Revenue & Customs
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- *@
-
-@import controllers.declaration.routes._
-@import forms.declaration. CommodityMeasure
-@import uk.gov.hmrc.play.views.html._
-@import views.Title
-@import models.requests.JourneyRequest
-@import controllers.navigation.{Navigator, ItemId}
-
-@this(main_template: views.html.main_template)
-
-@(mode: Mode, itemId: String, form: Form[CommodityMeasure])(implicit request: JourneyRequest[_], messages: Messages)
-
-@tariffBodyDesc = { @Html(messages("declaration.type.consignmentTariffText", components.details_content_link())) }
-
-@tariffBody = { @components.tariff_body_partial(messages("declaration.type.commodityMeasureListItemOne"), messages("declaration.type.commodityMeasureListItemTwo"), messages("declaration.type.commodityMeasureListItemThree"))(tariffBodyDesc)}
-
-@main_template(
```

Comment:
Is it some unused page?

Reply:
Probably we should be able in that case clear the messages that are not used after removing the file

Reply:
Yes, it's not used.  I was looking for pages that were still not using GDS.  Good point about the messages.  I'll do that now.

Reply:
I should read the commit messages to not ask stupid questions 😂

Reply:
It's early.  And Friday!

Reply:
Just checked - there are no messages referenced in this file that aren't either being used elsewhere or not in the messages.en file (i.e. already purged)